src/malloc_wrapper.cc

```diff
@@ -99,7 +99,7 @@ void *__wrap_malloc(size_t size)
 {
 	void *lptr = __real_malloc(size);
```

Comment:
[llvmlibc-callee-namespace]: 'info<int>' must resolve to a function declared within the '__llvm_libc' namespace